org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/LanguageServerImpl.xtend

```diff
@@ -134,7 +143,7 @@ import org.eclipse.xtext.validation.Issue
 		]
 
 		requestManager.runWrite [ cancelIndicator |
-			val rootURI = URI.createFileURI(params.rootPath).toPath.toUri
+			val rootURI = URI.createURI(rootPath.toUri.toString)
```

Comment:
IMO it does so - I just condensed the repeated 'toPath()' & 'toUri()' calls. I'm convinced that 'Paths.get().toUri()' does the correct thing. We just need to be aware that 'Path.get()' may behave slightly different on different platforms (I remember a situation where my code worked well on my Mac but didn't on a Windows system)